server/views/pages/debug/reportDetails.njk

```diff
@@ -2,138 +2,246 @@
 
 {% from "govuk/components/table/macro.njk" import govukTable %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{%- from "govuk/components/tag/macro.njk" import govukTag -%}
 
 {% set pageTitle = applicationName + " - Report" %}
 
+{% block beforeContent %}
+  {{ govukBackLink({
+  text: "Back",
+  href: "/reports"
+  }) }}
+{% endblock %}
+
+{%- set location =  prisonsLookup[report.location].description or report.location -%}
+{%- set reportingOfficer =  usersLookup[report.reportedBy].name or report.reportedBy -%}
+
 {% block content %}
-  <h1>Report '{{ report.title }}'</h1>
+  <h1 class="govuk-heading-m">Report {{ report.reportReference }}
+    {{ govukTag({
+    text: report.status
+    }) }}
+  </h1>
+  <h2 class="govuk-heading-s">Incident reported in {{ location }} by {{ reportingOfficer }}</h2>
 
-  {{ govukTable({
-  caption: "Report details",
-  captionClasses: "govuk-table__caption--m",
-  firstCellIsHeader: true,
+<!-- Incident details table -->
+{{ govukSummaryList({
+  card: {
+    title: {
+      text: "Incident details"
+    },
+    actions: {
+      items: [
+        {
+          href: "/reports/" + report.id + "/update-details",
+          text: "Change",
+          visuallyHiddenText: "incident details"
+        }
+      ]
+    }
+  },
   rows: [
-    [
-      {
-        text: "Type"
-      },
-      {
-        text: report.type
-      }
-    ],
-    [
-      {
-        text: "Title"
-      },
-      {
-        text: report.title
-      }
-    ],
-    [
-      {
-        text: "Reference"
-      },
-      {
-        text: report.reportReference
-      }
-    ],
-    [
-      {
-        text: "Status"
-      },
-      {
-        text: report.status
-      }
-    ],
-    [
-      {
-        text: "Location"
-      },
-      {
-        text: prisonsLookup[report.location].description or report.location
-      }
-    ],
-    [
-      {
-        text: "Incident time"
-      },
-      {
-        text: report.incidentDateAndTime | dateAndTime
-      }
-    ],
-    [
-      {
-        text: "When reported"
+    {
+      key: {
+        text: "Date and time of incident"
       },
-      {
-        text: report.reportedAt | dateAndTime
+      value: {
+        html: report.incidentDateAndTime | dateAndTime
       }
-    ],
-    [
-      {
-        text: "Reported by"
+    },
+    {
+      key: {
+        text: "Description"
       },
-      {
-        text: usersLookup[report.reportedBy].name or report.reportedBy
+      value: {
+        html: report.description
```

Comment:
it's safer to have `text: x` in these items!
(so that something like `<script>alert('lol')</script>` in the description doesn't rickroll users 😉)

Reply:
there are a few more `html: x` below that would also be safer as `text: x`